### ISSUE TITLE
Simple-History doesn't remember settings with which media was watched (speed, audio filters, etc).

### ISSUE THREAD

**Issue by kanjieater** [Eisa01/mpv-scripts#111]
Sometimes I want to watch different content at different speeds (for me this is based on complexity - if somethings difficult to understand I prefer to watch it at 1x speed). Being able to remember the playback speed like MPV does as well as other settings like audio filters is a useful feature on a per content basis.

Current Behavior:
1. Watch media A at 1.2x speed.
2. Pull up media B through simple history.
3. Media B is played at 1.2x speed.

Normal MPV behavior
1. Open Media A in a new mpv window. 
2. Media A played at whatever the last speed was set to for Media A
3. Open Media B
4. Media B played at whatever the last speed was set to for Media B

Desired Behavior:
Each media remember's its own settings, and when you switch between them with simple-history, it remembers what settings it should be playing with.

**Comment by Eisa01** [comment]
Closed as duplicate of #84 , 

Basically I implemented it in the updated LogManager  (currently only for SimpleBookmakr) but I didn't have time to migrate it to the other scripts.

It will be implemented, but currently I dont have specific timelime since I am too busy :/

**Comment by kanjieater** [comment]
Ok - so if I load SimpleBookmarker as well would this fix it?

**Comment by Eisa01** [comment]
No this feature needs to be migrated, I need to add LogManager that is inside SimpleBookmark to SimpleHistory.
Once that is done, it will be possible to enable this option.
